File: logica-general/ejercicio-019/resoluciones/lucas-pajarito/formulas.js
const reactivos = [
    {
        nombre: "Ácido Clorhídrico",
        cantidad: 8,
        minimo: 5
    },
    {
        nombre: "Hidróxido de Sodio",
        cantidad: 3,
        minimo: 5
    },
    {
        nombre: "Sulfato de Cobre",
        cantidad: 10,
        minimo: 6
    },
    {
        nombre: "Etanol",
        cantidad: 2,
        minimo: 4
    },
    {
        nombre: "Agua Destilada",
        cantidad: 15,
        minimo: 10
    }
];

function revisarInventario(reactivos) {

    reactivos.forEach(reactivo => {

        if (reactivo.cantidad >= reactivo.minimo) {
            console.log(`${reactivo.nombre}: Stock suficiente.`);
        } else {
            console.log(`${reactivo.nombre}: Reabastecer inventario.`);
        }

    });

}

revisarInventario(reactivos);

File: logica-general/ejercicio-039/resoluciones/lucas-pajarito/formulas.py
reactivos = [
    {
        "nombre": "Ácido Clorhídrico",
        "cantidad": 12,
        "stock_minimo": 8
    },
    {
        "nombre": "Hidróxido de Sodio",
        "cantidad": 5,
        "stock_minimo": 6
    },
    {
        "nombre": "Etanol",
        "cantidad": 15,
        "stock_minimo": 10
    },
    {
        "nombre": "Sulfato de Cobre",
        "cantidad": 3,
        "stock_minimo": 5
    },
    {
        "nombre": "Agua Destilada",
        "cantidad": 20,
        "stock_minimo": 10
    }
]


def revisar_inventario(reactivos):

    print("=== INVENTARIO DE REACTIVOS ===")

    for reactivo in reactivos:

        print(f'Reactivo: {reactivo["nombre"]}')
        print(f'Cantidad disponible: {reactivo["cantidad"]}')
        print(f'Stock mínimo: {reactivo["stock_minimo"]}')

        if reactivo["cantidad"] >= reactivo["stock_minimo"]:
            print("Estado: Inventario suficiente.")

        else:
            print("Estado: Es necesario reabastecer el inventario.")

        print("--------------------------------------")


revisar_inventario(reactivos)